bin/cqlsh.py

```diff
@@ -1881,6 +1881,21 @@ def do_clear(self, parsed):
         subprocess.call('clear', shell=True)
     do_cls = do_clear
 
+    def do_history(self,parsed):
+        """
+        history/HIS
+
+        Displays the most recent (up to 50) commands executed in cqlsh.
+        """
+        recent_history = ""
+        hisotry_length = readline.get_current_history_length()
+        for index in range(hisotry_length, hisotry_length-50, -1):
```

Comment:
maybe make `hisotry_length-50` configurable as an environment variable

Reply:
What about making history_length-50 configurable via an optional input parameter instead? This way, the user doesn't have to re-configure an environmental variable every time they want to view more/less. Let me know what you think about the input parameter approach vs the environmental variable approach!

Reply:
I think making the history length configurable via optional input parameter makes sense. We'd still want a configuration for the default length (ie. 50) when the parameter is not specified. This could be an environmental variable or a [cqlshrc](https://docs.datastax.com/en/dse/5.1/cql/cql/cql_reference/cqlsh_commands/cqlshCqlshrc.html) configuration. Perhaps we can start with an environment variable and later expand to allow the default size to be specified in `cqlshrc` ?